languages/go/exercises/concept/strings/go.mod

```diff
@@ -0,0 +1,3 @@
+module exercism/go/concept/strings
```

Comment:
what do you think about this module scheme?

Reply:
Generally in my experience I would rather do something more like github.com/mcaci/... so ending up here being github.com/exercism/v3/languages/go/exercises/concept/strings which however sounds a bit long so not really ideal. But what's the idea behind introducing modules? Do we want the exercises to be go gettable?

Reply:
@mcaci: We had to add `go modules` to all the exercises in `v2` so the tests can be executed locally. `go test` does not run outside of `GOPATH` any more since the introduction of `go modules` if there is no module initiated. If you check the exercises in https://github.com/exercism/go/tree/master/exercises, they all have a `go.mod` file.

I just checked and they all have `module <SLUG>`, no extra path.

Maybe just do `module concept/<SLUG>` or just `module <SLUG>` as in v2?

Reply:
Will use the same as in `v2` for now.

Reply:
Thanks for the explanation, you're absolutely correct. Ok for the module name for me :)